docs/installation.md

```diff
@@ -291,12 +289,12 @@ unzip it, and then run:
 ./refresh-data.sh /path/to/dump.sql
 ```
 
-This will remove the initial Wagtail admin user that was created by
-the `initial-data.sh` script that was called by `backend.sh`.
-If you need to access the Wagtail admin, create a new user with the following:
+The `refresh-data.sh` script will apply the same changes as the `initial-data.sh` script described above (including setting up the `admin` superuser), but will not apply migrations.
```

Comment:
I had to stop and consider *why* refresh-data.sh wouldn't apply migrations and initial-data.sh would. It might be good to expand on this sometime, but that seems out of scope here. I'm just making a note of it so it's out of my head and into the PR record.